ipyleaflet/leaflet.py

```diff
@@ -969,7 +970,7 @@ class Map(DOMWidget, InteractMixin):
     max_zoom = CFloat(18).tag(sync=True, o=True)
     min_zoom = CFloat(1).tag(sync=True, o=True)
     interpolation = Unicode('bilinear').tag(sync=True, o=True)
-    crs = Enum(values=allowed_crs, default_value='EPSG3857').tag(sync=True)
+    crs = Dict(default_value=projections.EPSG3857).tag(sync=True, o=True)
```

Comment:
You should not add  the `o=True`, because the `crs` parameter is treated differently than other options JavaScript side.